.github/workflows/ci.yml

```diff
@@ -13,7 +13,16 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  authorize:
+    environment:
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external-contributors' || 'internal-contributors' }}
```

Comment:
```suggestion
      'external-contributors'}}
```

Reply:
Nope, that shouldn't be the cause because that environment doesn't require so.
![CleanShot 2025-02-27 at 12 29 40](https://github.com/user-attachments/assets/d273548d-b82d-4a16-83ba-c6810b0d9292)